bin/server/cmd/root.go

```diff
@@ -131,6 +144,12 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&flags.demoName, "demo-name", "", "name of the demo to use when running in demo mode")
 	rootCmd.PersistentFlags().BoolVar(&flags.debug, "debug", false, "whether to enable debug level logging")
 	rootCmd.PersistentFlags().StringVar(&flags.pgURL, "pg", "", "optional external PostgreSQL instance connection url(must provide dbname); for example postgresql://user:secret@masterhost:5432/dbname?sslrootcert=cert")
+
+	// Cloud backup related flags.
+	// TODO(dragonly): Add GCS usages when it's supported.
+	rootCmd.PersistentFlags().StringVar(&flags.backupRegion, "backup-region", "", "region of the backup bucket, e.g., us-west-2 for AWS S3.")
+	rootCmd.PersistentFlags().StringVar(&flags.backupBucket, "backup-bucket", "", "bucket where Bytebase stores backup data, e.g., s3://example-bucket. When provided, Bytebase will store data to the S3 bucket.")
+	rootCmd.PersistentFlags().StringVar(&flags.backupCredentials, "backup-credentials", "", "credentials file to use for the backup bucket. It should be the same format as the AWS credential files.")
```

Comment:
I would choose singular form: backup-credential

1. It's a single file containing a single credential object
2. Plural is hard anyway